backend/apps/ollama/main.py

```diff
@@ -724,7 +726,7 @@ async def generate_chat_completion(
     }
 
     model_id = form_data.model
-    model_info = Models.get_model_by_id(model_id)
+    model_info = Models.get_model_by_id(db, model_id)
```

Comment:
Ideally we wouldn't want to modify the router codes to include the db param if possible.